tests/graphql/errors/test-const.graphql

```diff
@@ -0,0 +1,17 @@
+#> server-sdl
+schema @server(baseURL: "https://jsonplaceholder.typicode.com") {
+    query: Query
+}
+
+type User {
+    name: String
+    age: Int
+}
+
+type Query {
+    user: User @const(data: {name: "John"})
```

Comment:
This should only fail if age is a required feild

Reply:
Covered this comment in new commit.